server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java

```diff
@@ -123,7 +123,7 @@
   public static final ConfigOption<Long> SERVER_PRE_ALLOCATION_EXPIRED = ConfigOptions
       .key("rss.server.preAllocation.expired")
       .longType()
-      .defaultValue(10 * 1000L)
+      .defaultValue(120 * 1000L)
```

Comment:
120 is too long for preAllocation expired

Reply:
In sop and online ,  we choose 120s. What's your suggestion?

Reply:
when client send data to shuffle server, request memory first, then send data if there has enough memory. Shuffle server will keep the preAllocation memory until preAllocation.expired. If application is killed, there will be a lot of preAllocation memory in shuffle server.
preAllocation.expired is about interval of 2 rpc calls. If 10s is too short, it means there has some problem in rpc communication, maybe overload, etc.  These problem should be resolved first, preAllocation.expired is just a workaround for these problem.
I suggest to set 20s as default value at most

Reply:
OK